torch/ao/quantization/_dbr/quantization_state.py

```diff
@@ -117,6 +117,15 @@ def __init__(
         # to its final value after tracing.
         self.needs_dtype_transform_on_outputs = True
 
+    def get_extra_state(self):
+        return self.tensor_id_to_scale_zp
```

Comment:
might be valuable to define this as a dictionary, `{'tensor_id_to_scale_zp': self.tensor_id_to_scale_zp}`.  This way if a future PR needs to add more things here, it can be done in a backward compatible way.